dev_guide/pod_autoscaling.adoc

```diff
@@ -215,6 +230,21 @@ spec:
 $ oc create -f hpa.yaml
 ----
 
+[IMPORTANT]
+====
+For memory-based autoscaling to work, memory usage must increase and decrease
+proportionally to the replica count. On average:
+
+* An increase in replica count must lead to an overall decrease in memory
+(working set) usage per-pod.
+* A decrease in replica count must lead to an overall increase in per-pod memory
+usage.
+
+Use the OpenShift dashboard to verify the memory behavior of your application
+and ensure that your application meets these requirements before using
+memory-based autoscaling.
+====
```

Comment:
Do we need to tell users how to ensure the memory behavior?

Reply:
Thanks @mburke5678, I am updating this to `.. use the web console to check the memory ...` I think this refers to manually increasing the replicas in web console and look at the memory graph.